system_testing/test_bucky_project/services/service_a/pc/verifications/index.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'bucky/test_equipment/verifications/e2e_verification'
+module Services
+  module ServiceA
+    module Pc
+      module Verifications
+        class Index < Bucky::TestEquipment::Verifications::E2eVerification
+          def click_single_element(_)
+            @pages.index.links.click
```

Comment:
I think it is available to operate the element in verification.
But In specification, can we make the verify method without using assert?